tests/functional/test_zilien_parser.py

```diff
@@ -0,0 +1,146 @@
+"""This module perform top-level finctional tests on the Zilien parsers"""
```

Comment:
Can you please delete this file?
This file is a partial duplicate of "test_zilien_reader.py" - something went wrong with git's tracking of its rename.